### PR TITLE
remove ref to travis

### PR DESCRIPTION
**Pull request by jburel**
Remove ref noticed while preparing workshop
cc @pwalczysko

### CODE REVIEW

File: write_guide.rst [hunk 1]
```diff
@@ -5,9 +5,11 @@ The OMERO guide is hosted on `readthedocs <https://readthedocs.org/>`_.
...
 To create a new guide:
-  - create a new repository on GitHub using the `template guide <https://github.com/ome/guide-template>`_
-  - enable travis to it is automatically built when commits are pushed
-  - add the new repository as a new project on `readthedocs <https://readthedocs.org/>`_
-  - add the new repository as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure
-  - add it to the this repository as a submodule using the same alias.
-  - insert the guide into one of the existing files or in a new file it is a new section.
+  - Create a new repository on GitHub using the `template guide <https://github.com/ome/guide-template>`_.
+  - Add the new repository as a new project on `readthedocs <https://readthedocs.org/>`_.
+  - Add the new repository as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure,
```
Comment (pwalczysko) [inline write_guide.rst:10]:
```suggestion
  - Add the new project as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure,
```

File: write_guide.rst [hunk 2]
```diff
@@ -5,9 +5,11 @@ The OMERO guide is hosted on `readthedocs <https://readthedocs.org/>`_.
...
+  - Add it to the this repository as a submodule using the same alias.
```
Comment (pwalczysko) [inline write_guide.rst:11]:
```suggestion
  - Add the new repository to the `omero-guides repository <https://github.com/ome/omero-guides>`_ as a submodule using the same alias.
```

File: write_guide.rst [hunk 3]
```diff
@@ -5,9 +5,11 @@ The OMERO guide is hosted on `readthedocs <https://readthedocs.org/>`_.
...
+  - Insert the guide into one of the existing files or in a new file it is a new section.
+  - If the new repository will contain a ``binder`` folder to generate an environment for analysis,
```
Comment (pwalczysko) [inline write_guide.rst:13]:
```suggestion
  - If you intend that the new repository contains a ``binder`` folder to generate an environment for analysis,
```

File: write_guide.rst [hunk 4]
```diff
@@ -5,9 +5,11 @@ The OMERO guide is hosted on `readthedocs <https://readthedocs.org/>`_.
...
+  add to the `binder_badge.yml <https://github.com/ome/.github/blob/master/workflow-templates/binder_badge.yml>`_ and `repo2docker.yml <https://github.com/ome/.github/blob/master/workflow-templates/repo2docker.yml>`_ workflows to ``.github/workflows``.
```
Comment (pwalczysko) [inline write_guide.rst:14]:
```suggestion

```

Removing this sentence completely, as I suggest to add it to the previous bullet point to fix formatting, see https://github.com/ome/omero-guides/pull/116/files#r636809563

**Comment by pwalczysko** [comment]
Couple of suggestions, please check if I got the meaning right.

**Comment by jburel** [comment]
Thanks @pwalczysko

**Comment by pwalczysko** [comment]
How about https://github.com/ome/omero-guides/pull/116#discussion_r636804830 and https://github.com/ome/omero-guides/pull/116#discussion_r636806997 ?

**Comment by jburel** [comment]
done

File: write_guide.rst [hunk 5]
```diff
@@ -5,9 +5,9 @@ The OMERO guide is hosted on `readthedocs <https://readthedocs.org/>`_.
...
-  - enable travis to it is automatically built when commits are pushed
-  - add the new repository as a new project on `readthedocs <https://readthedocs.org/>`_
-  - add the new repository as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure
-  - add it to the this repository as a submodule using the same alias.
-  - insert the guide into one of the existing files or in a new file it is a new section.
+  - Create a new repository on GitHub using the `template guide <https://github.com/ome/guide-template>`_.
+  - Add the new repository as a new project on `readthedocs <https://readthedocs.org/>`_.
+  - Add the new project as a subproject of `omero-guides <https://readthedocs.org/projects/omero-guides/>`_ using an alias e.g. omero-guide-figure has been added as figure,
+  - Add the new repository to the `omero-guides repository <https://github.com/ome/omero-guides>`_ as a submodule using the same alias.
+  - Insert the guide into one of the existing files or in a new file it is a new section.
```
Comment (pwalczysko) [inline write_guide.rst:12]:
```suggestion
  - Insert the guide into one of the existing files or in a new file if it is a new section.
```

**Comment by pwalczysko** [comment]
Looks good to me, merging.

**Pull request merged**